eac workaround

mactan-sc opened the pull request:
ngh was poking around the json workaround option and found this variable was now available with the updated eac SDK and a few of us tried it out and it worked fine

thought it might be time to remove some of the old dxvk and wine sources until needed again

the-sane commented:
Oh cool, so this effectively disables EAC?

Do we still need to delete the EAC directory when using this or can we get rid of that prelaunch command too?

mactan-sc commented:
Indeed, I removed my prelaunch script from lutris as well and it continues to work with just the env variable

the-sane commented:
Could you also please check what happens if the new variable is removed, the game is launched (so all the eac stuff downloads), and then the new variable is re-added without deleting the eac directories? Just to double confirm that nobody will run into any weird edge case scenarios.

mactan-sc commented:
- I removed the variable and launched the game, got code 4 EAC violation
- added the variable back, no further actions, and relaunched. successfully got into the PU

I played the game with it for about 40 minutes the other day to help affirm there wasn't a kick over longer duration

the-sane commented:
Excellent, thanks!
